src/EnergyPlus/HighTempRadiantSystem.cc

```diff
@@ -451,6 +451,7 @@ namespace HighTempRadiantSystem {
             }
 
             if (state.dataIPShortCut->lAlphaFieldBlanks(7)) {
+                highTempRadSys.setptSched = Sched::GetScheduleAlwaysOff(state);
```

Comment:
Empty field means "AlwaysOff" schedule, so initialize with pointer to "AlwaysOff" schedule object. Not sure how "AlwaysOff" is a setpoint exactly, but that was the previous behavior. There could be a few more of these stragglers and this is the fix for all of them.

Reply:
Seems like it's dependent on the "Temperature control type" field.

```
    MeanAirTemperature
    MeanRadiantTemperature
    OperativeTemperature
    MeanAirTemperatureSetpoint
    MeanRadiantTemperatureSetpoint
    OperativeTemperatureSetpoint
```

One could argue that it should probably fatal out if the ControlType is a Setpoint one and the schedule is missing, but I think it just makes the HighTempRadiant to never do anything,

https://github.com/NREL/EnergyPlus/blob/09a25d0d10d3a69ef2dc1bb8b45a82d898bed64a/src/EnergyPlus/HighTempRadiantSystem.cc#L915

Reply:
If one can argue that it should fatal out than I will make it fatal out. If one can argue that this should be a warning that the system will be disabled then I will do that. I would like to handle this the "right" way. Silently disabling this system if the schedule is empty does not seem like the "right" way.

Reply:
> If one can argue that it should fatal out than I will make it fatal out. If one can argue that this should be a warning that the system will be disabled then I will do that. I would like to handle this the "right" way. Silently disabling this system if the schedule is empty does not seem like the "right" way.

@RKStrand ?

Reply:
The schedule is also used if it's not a SetpointControl temperature control type.

I/O Ref states this:
> Field: Heating Setpoint Temperature Schedule Name[[LINK]](https://bigladdersoftware.com/epx/docs/24-2/input-output-reference/group-radiative-convective-units.html#field-heating-setpoint-temperature-schedule-name-1)

> This field specifies the heating setpoint or control temperature for the radiant system in degrees Celsius.

https://github.com/NREL/EnergyPlus/blob/09a25d0d10d3a69ef2dc1bb8b45a82d898bed64a/src/EnergyPlus/HighTempRadiantSystem.cc#L804

One could draw the conclusion that the E+ IDD should probably just make this schedule `\required-field`.

Reply:
TemperatureControlType (A6) is also not a required field. And when it is left empty, there are is no error message but there are asserts.